usda_nutrition/management/commands/import_usda.py

```diff
@@ -1,6 +1,11 @@
 import csv
 import os
 import sys
+if (sys.version_info < (3, 0)):
+    reload(sys)
+    sys.setdefaultencoding('utf-8')
```

Comment:
Oh...I also tested it out on python3 and the import script still works just fine.